stylus/components/button.styl

```diff
@@ -0,0 +1,30 @@
+// Buttons of any kind
+.coz-btn
+    box-sizing border-box
+    display inline-block
+    vertical-align top
+    margin 0 em(8px)
+    border 1px solid grey-09
+    height: em(43px)
+    padding em(11px) em(15px) em(10px)
+    background-color white
+    font-size(14px)
+    text-transform uppercase
+
+    &--regular
+        border-color blue2
+        background-color blue2
+        color white
+
+    &--upload
+        left-padded(44px)
+        background-image: embedurl("../../node_modules/cozy-ui/assets/icons/icon-upload.svg")
```

Comment:
Ok at this time for the path, I'l update the module config to add the cozy-ui assets folder in the [lookup paths for the `embedurl` method](http://stylus-lang.com/docs/functions.url.html) so referencing external file will be more convenient 👍